IdokladSdk.UnitTests/Tests/Validation/IbanAttributeTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿using IdokladSdk.Validation.Attributes;
+using NUnit.Framework;
+
+namespace IdokladSdk.UnitTests.Tests.Validation
+{
+    [TestFixture]
+    public class IbanAttributeTests
+    {
+        internal IbanAttribute IbanAttribute { get; set; }
```

Comment:
Proč je to internal a ne private readonly?

Reply:
opraveno.